### ISSUE TITLE
[BUG] Accessibility -App-  Drag and Drop functionality not accessible

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#1971]
**Describe the accessibility issue**
Some people cannot perform dragging movements in a precise manner. Others use a specialized or adapted input device such as a head pointer, eye-gaze system, or speech-controlled mouse emulator, which makes dragging cumbersome, error-prone, or outright impossible.
When an interface implements functionality that uses dragging movements, users perform four discrete actions:

1. tap or click to establish a starting point, then
2. press and hold that contact while...
3. performing a repositioning of the pointer, before...
4. releasing the pointer at the end point.

Not all users can accurately press and hold that contact while also repositioning the pointer. An alternative method must be provided so that users with mobility impairments who use a pointer (mouse, pen, or touch contact) can use the functionality.
[WCAG 2.5.7 Dragging Movements](https://www.w3.org/WAI/WCAG22/Understanding/dragging-movements)

**To Reproduce**

1. Go to 'Customize Favorites' screen

**Expected behavior**
Need an alternative to the drag/drop functionality. Possibly addition of up/down arrows so that users with pointers may access. 
Not all users can accurately press and hold that contact while also repositioning the pointer. An alternative method must be provided so that users with mobility impairments who use a pointer (mouse, pen, or touch contact) can use the functionality.

**Screenshots**
![IMG_0049](https://user-images.githubusercontent.com/82892873/178583095-e0c8e506-a68c-43c0-96c1-da41ee766360.PNG)

Smartphone (please complete the following information):
iPhone XR- iOS 14.6,

Additional context

**Comment by TodorBachvarov** [comment]
Implemented drag and drop with swipe up/down action which also contains functionality to move the selected item with the volume up/down buttons. Implemented in version 4.1.5

**Comment by acovher2** [comment]
https://user-images.githubusercontent.com/82892873/224023295-17671915-0f2f-4637-ab5e-3887e3f8fc07.mp4

Retest failed in 4.2.54. Please advise. The instruction given does not work with the screen reader on.

**Comment by acovher2** [comment]
@pmarkhennessy  this is still failing in 6.0.6 apk. Please advise. 
@knguyenjahiel

**Comment by acovher2** [comment]
@pmarkhennessy Due to the ADA title II changes and deadline of April 2026, I wanted to ensure this gets on the list and radar as we need to be compliant. This was on the PPT I sent you.

**Comment by pmarkhennessy** [comment]
@TodorBachvarov Can you re-visit this please

**Comment by acovher2** [comment]
@TodorBachvarov Yes the up/down motions are working. There needs to be something in place at the top and bottom of list to let user know they are done going up or down. Currently it is just a "bbmp" sound nothing in wording saying top of list or bottom of list.

**Comment by acovher2** [comment]
@TodorBachvarov 
iOS iPhone 14+ ---> When I use VoiceOver on this feature, iOS is not functioning the same as Android. First, there is no instruction such as "Double tab and hold to move" like there is on Android. Second, no matter what I do, I cannot get them to drag on iOS. Please advise. 

Android 14 with TalkBack I am able to drag and this alternative is working to a point. The dragging motion can still be difficult for some users. I believe the TARC would probably issue an exception to us if we could get this working on iOS too.

**Comment by TodorBachvarov** [comment]
@acovher2  You should focus the handle icon, both IOS and android support drag and drop with the same gesture (double tap and hold, after that drag). 
In version 7.07 I've updated the hint for IOS and also added alternative way to move the items. Now the handle icon is focusable and support the double tap and hold functionality also the whole item is focusable and it's marked as slider. In this way you should slide up or down and the item will be moved with one position up or down. I've added the relevant hints as well. Please let me know if this will be enough.

**Comment by TodorBachvarov** [comment]
@acovher2 The Drag and drop functionality was improved in version 7.2.0. 
The drag handle is now focusable on both ios and android. (double tap and hold, after that drag)
Slide gesture was added as alternative method for reordering. (slide up/down)

**Comment by vburgett** [comment]
@acovher2 can you test this? Thank you!

**Comment by acovher2** [comment]
> 3\. I've fixed the focus of the Star icon, now it's focusable and the user can add / remove items from favourite easier (the swipe up/down functionality can be used for reordering).

@TodorBachvarov in 7.3.51 this functionality has changed. I'm no longer able to drag and drop by swiping up/down.   Reopening this issue. Please advise.

**Comment by acovher2** [comment]
@TodorBachvarov I thought this wasn't working but it seems it only moves one position at a time after original instruction (without needing to double tap) , then it is as if it loses focus and then requires the double tap to continue re-ordering the item. I will open a separate ticket for this different issue.
